scalalib/overrides-2.11/scala/Array.scala

```diff
@@ -0,0 +1,531 @@
+/*                     __                                               *\
```

Comment:
The Array::copy method was replaced.  

Original:

```scala
  def copy(src: AnyRef, srcPos: Int, dest: AnyRef, destPos: Int, length: Int) {
    val srcClass = src.getClass
    if (srcClass.isArray && dest.getClass.isAssignableFrom(srcClass))
      arraycopy(src, srcPos, dest, destPos, length)
    else
      slowcopy(src, srcPos, dest, destPos, length)
  }
```

New

```scala
  def copy(src: AnyRef, srcPos: Int, dest: AnyRef, destPos: Int, length: Int) {
    arraycopy(src, srcPos, dest, destPos, length)
  }
```

It is necessary, because `java.lang.Class` isn't implemented yet. 

But even when `java.lang.Class` will be implemented, the whole `slowcopy` logic seems to solve a JVM problem.  So, IMO, the `scala.Array::copy` method should always call `scala.compat.Platform::arraycopy` and platforms specific details should be resolved in the latter method.